src/InstagramClone.Api/Startup.cs

```diff
@@ -13,26 +15,47 @@ public static WebApplication InitialIzeApp(string[] args)
             Configure(app);
             return app;
         }
-       
+
 
         private static void ConfigureService(WebApplicationBuilder builder)
         {
             
+            builder.Services.AddControllers(config =>
+            {
+                config.RespectBrowserAcceptHeader = true;
+                config.ReturnHttpNotAcceptable = true;
+            }).AddXmlDataContractSerializerFormatters();
+
+             builder.Services.AddControllers();
```

Comment:
something is wrong here and I have given you similar feedback about duplicates recently. 

fix up

Reply:
this is the 2nd time you're calling `builder.Services.AddControllers();` 

the 1st time above has some configuration parameters, and here you call it without any parameters. can you explain the difference?

why do you have it twice?